build.gradle

```diff
@@ -0,0 +1,74 @@
+plugins {
+    id 'application'
+    alias(libs.plugins.springboot)
+}
+
+repositories {
+    mavenCentral()
+    maven { url 'https://jitpack.io' }
+}
+
+group 'bisq'
+
+apply plugin: "org.springframework.boot"
+apply plugin: 'io.spring.dependency-management'
+
+version = file("src/main/resources/version.txt").text.trim()
+
+application {
+    project.mainClassName = 'bisq.price.Main'
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(11))
+    }
+}
+
+compileJava {
+    options.release = 11
+}
+
+jar.manifest.attributes(
+        "Implementation-Title": project.name,
+        "Implementation-Version": version
+)
+
+configurations.all {
```

Comment:
There will only be one project.   I did it this way to not have to avoid repeating any exclusions in separate implementation 'lib' configurations.

While we're on the subject of exclusions...  The IDE is showing many duplicate dependencies (same lib, different version) in it's External Libraries list. I can't seem to control what it displays.   (I see this problem in the Bisq2 project too.)  Could it be that Intellij's gradle support has not caught up to the new version catalog feature?

I don't worry about it too much.  Besides being a bit annoying, it might force you to choose a library to step into during debugging.

The lib dir created in the gradle distribution looks correct -- adhering to version constraints and exclusions.  So do the gradle complie and run time dependencies tree displayed in the Gradle panel in the upper right of the IDE.